Makefile

```diff
@@ -71,15 +78,20 @@ check: python-env
 # Corrects spelling errors
 .PHONY: misspell
 misspell:
-	go get github.com/client9/misspell
-	${FIND} -name '*'  -exec misspell -w {} \;
+	go get $(MISSPELL_REPO)
+	$(FIND) -name '*'  -exec $(MISSPELL) -w {} \;
 
 .PHONY: fmt
 fmt: python-env
 	$(foreach var,$(PROJECTS),$(MAKE) -C $(var) fmt || exit 1;)
 	# Cleans also python files which are not part of the beats
 	find . -type f -name *.py -not -path "*/vendor/*" -not -path "*/build/*" -not -path "*/.git/*" -exec autopep8 --in-place --max-line-length 120  {} \;
 
+.PHONY: lint
```

Comment:
I thought it would be nice to mimic what Jenkins does, i.e., run it on the diff. Are you suggesting running it on everything by default or having the ability to restrict it to the diff of a single beat?